docs/docs/cmd/spo/list/list-defaultvalue-remove.mdx

```diff
@@ -0,0 +1,62 @@
+import Global from '/docs/cmd/_global.mdx';
+
+# spo list defaultvalue remove
+
+Removes a specific default column value for a specific document library
+
+## Usage
+
+```sh
+m365 spo list defaultvalue remove [options]
+```
+
+## Options
+
+```md definition-list
+`-u, --webUrl <webUrl>`
+: URL of the site where the list is located.
+
+`-i, --listId [listId]`
+: ID of the list. Specify either `listTitle`, `listId`, or `listUrl`.
+
+`-t, --listTitle [listTitle]`
+: Title of the list. Specify either `listTitle`, `listId`, or `listUrl`.
+
+`--listUrl [listUrl]`
+: Server- or site-relative URL of the list. Specify either `listTitle`, `listId`, or `listUrl`.
+
+`--fieldName <fieldName>`
+: Internal name of the field.
+
+`--folderUrl [folderUrl]`
+: Remove field from a specific folder. By default, the root folder of the list is used.
```

Comment:
Should we rename this to make it clearer what type of URL needs to be provided?


```suggestion
: Server-relative URL of a specific folder to remove the field from. By default, the root folder of the list is used.
```

Reply:
It should work both with a server-relative URL and a site-relative URL.

Reply:
yes, but someone might think a library-relative folder can be used as well. Which is why I'd vote for being clear about it;

```ts
Server- or site-relative URL of a specific folder to remove the field from. By default, the root folder of the list is used.
```